### PR TITLE
(enhancement) update patient-banner config-schema

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

This is a follow-up PR to correct patient-banner config-schema

### CODE REVIEW

File: packages/esm-patient-banner-app/src/config-schema.ts [hunk 1]
```diff
@@ -2,13 +2,19 @@ import { Type } from '@openmrs/esm-framework';
...
 export const configSchema = {
   contactAttributeType: {
-    _type: Type.UUID,
+    _type: Type.Array,
     _description:
       'The Uuids of person attribute-type that captures contact information `e.g Next of kin contact details`',
+    _elements: {
+      attributeTypeUuid: {
+        _type: Type.UUID,
+        _description: 'Attribute type Uuid',
```
Comment (denniskigen) [inline packages/esm-patient-banner-app/src/config-schema.ts:11]:
```suggestion
        _description: 'The UUID of a specific person attribute type',
```

File: packages/esm-patient-banner-app/src/hooks/usePatientAttributes.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const usePatientContactAttributes = (patientUuid: string) => {
   const { contactAttributeType } = useConfig() as ConfigObject;
   const { attributes, isLoading } = usePatientAttributes(patientUuid);
   const contactAttributes = attributes.filter(({ attributeType }) =>
-    contactAttributeType?.some((uuid) => attributeType.uuid === uuid),
+    contactAttributeType?.some(({ attributeTypeUuid }) => attributeType.uuid === attributeTypeUuid),
```
Comment (denniskigen) [inline packages/esm-patient-banner-app/src/hooks/usePatientAttributes.tsx:37]: Suggests `contactAttributeType` is plural. Should it be?

**Review by denniskigen — COMMENTED** [review]
Thanks for the quick turnaround, @donaldkibet. I've added a few semantic suggestions.

**Comment by brandones** [comment]
Do we expect to add more properties to the object? Or can we just have `contactAttributeTypeUuids: ['abc-123', 'bdc-234']`?

**Comment by brandones** [comment]
@donaldkibet it looks like you just changed the Type from UUID to String. Why did you do that? What do you think of my suggestion?

**Comment by brandones** [comment]
(oops sorry wrong button)

**Comment by ZacButko** [comment]
@donaldkibet any updates on this?

**Comment by ZacButko** [comment]
Closing due to lack of activity

**Pull request closed without merging**